meilisearch/index.py

```diff
@@ -1637,6 +1637,124 @@ def reset_dictionary(self) -> TaskInfo:
 
         return TaskInfo(**task)
 
+    # TEXT SEPARATOR SUB-ROUTES
+
+    def get_separator_tokens(self) -> List[str]:
+        """Get the additional text separator tokens set on this index.
+
+        Returns
+        -------
+        settings:
+            List containing the separator tokens of the index.
+
+        Raises
+        ------
+        MeilisearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+        """
+        return self.http.get(self.__settings_url_for(self.config.paths.separator_tokens))
+
+    def get_non_separator_tokens(self) -> List[str]:
+        """Get the list of disabled text separator tokens on this index.
+
+        Returns
+        -------
+        settings:
+            List containing the disabled separator tokens of the index.
+
+        Raises
+        ------
+        MeilisearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+        """
+        return self.http.get(self.__settings_url_for(self.config.paths.non_separator_tokens))
+
+    def update_separator_tokens(self, body: List[str]) -> TaskInfo:
```

Comment:
You did this correctly according to what I told you, but it looks like the team wants it the other way so this change is my fault.

```suggestion
    def update_separator_tokens(self, body: Union[List[str], None]) -> TaskInfo:
```

Reply:
Yes, I saw the comment on the other PR only after submitting this one :( bad timing